nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -209,7 +213,7 @@ in
 
       authorizedKeysFiles = mkOption {
         type = types.listOf types.str;
-        default = [];
+        default = [ ".ssh/authorized_keys" ".ssh/authorized_keys2" "/etc/ssh/authorized_keys.d/%u" ];
```

Comment:
See https://github.com/NixOS/nixpkgs/pull/10155 for how this works; any values set in `authorizedKeysFiles` are merged into the default, and I don't think we are likely to change this behavior soon due to backwards compatability, so please undo this change. However, since this makes at least 2 people confused by this, I think a documentation update would be great!